project/Win32BuildSetup/BuildSetup.bat

```diff
@@ -56,61 +55,18 @@ FOR %%b in (%1, %2, %3, %4, %5, %6) DO (
   IF %%b==nomingwlibs SET buildmingwlibs=false
   IF %%b==nobinaryaddons SET buildbinaryaddons=false
   IF %%b==sh SET useshell=sh
-  IF %%b==cmake SET cmake=1
 )
 
 SET buildconfig=Release
-set WORKSPACE=%CD%\..\..
+set WORKSPACE=%CD%\..\..\kodi-build
```

Comment:
you forgot to cutoff some kodi-build

set EXE="%WORKSPACE%\kodi-build\%buildconfig%\%APP_NAME%.exe"
set PDB="%WORKSPACE%\kodi-build\%buildconfig%\%APP_NAME%.pdb"
set D3D="%WORKSPACE%\kodi-build\D3DCompile*.DLL"